TAO/tao/Basic_Types_IDLv4.h

```diff
@@ -0,0 +1,58 @@
+#ifndef TAO_BASIC_TYPES_IDLV4_H
```

Comment:
Why not put these in `Basic_Types.h` and in the `CORBA` namespace, I think we should support `CORBA::Int8` without the requirement that the user includes this file, makes it much easier to use this construct which is also much clear in application code. For TAOX11 we could use this also without a special include and a namespace using.

Reply:
int8 is not a CORBA type (it has no Typecode, Any, IfR, DII, DSI, shouldn't be used for remote calls...).
For this whole group of type names, we wanted to make it explicit that these are different - tao_idl with default arguments will not accept them, they're not "peers" of the other names.

Reply:
Ok, we could keep Int8/UInt8 in this file, but the others are just typedefs which could be used without problems. That would enable TAOX11 to easily use these internally without a special include